internal/venice-client-common/src/main/java/com/linkedin/venice/httpclient5/HttpClient5Utils.java

```diff
@@ -83,11 +84,15 @@ public CloseableHttpAsyncClient buildAndStart() {
       final CloseableHttpAsyncClient client = HttpAsyncClients.customHttp2()
           .setTlsStrategy(tlsStrategy)
           .setIOReactorConfig(ioReactorConfig)
+          .setDefaultConnectionConfig(
```

Comment:
Great finding! How did you identify this? Is it because we didn't specify it in the past so a default and short timeout is used?